src/transformers/models/deit/modeling_deit.py

```diff
@@ -333,6 +337,7 @@ def forward(
         output_attentions=False,
         output_hidden_states=False,
         return_dict=True,
+        attention_mask=None,
```

Comment:
For some reason Github doesn't allow me to comment on the lines above, but one should pass the `attention_mask` after `hidden_states`.